docs/blog/pydantic-v2.md

```diff
@@ -210,7 +210,7 @@ In future direct validation of JSON will also allow:
     While in theory other formats could be specifically supported, the overheads and development time are
     significant and I don't think there's another format that's
     used widely enough to be worth specific logic. Other formats can be parsed to python then validated, similarly
-    when serialising, data can be exported to a python object, then serialised,
+    when serializing, data can be exported to a python object, then serialized,
```

Comment:
Just for the record, `serialising` is also correct, it's just a UK thing.

Reply:
yeah, I just wanted to standardize it, and since we use `serialize` more frequently than `serialise` throughout the codebase I just find-and-replaced everywhere